hangman.rb

```diff
@@ -0,0 +1,113 @@
+class Hangman
+
+  def initialize
+    @answer = pick_answer
+    @no_of_guesses = 0
+    @blanks_array = answer_to_blanks
+
+    start
+    puts @answer #temporary!!
+    art
+    get_guess
+
+  end
+
+  def start
+    puts "Welcome to Hangman!"
+  end
+
+  def pick_answer
+    # randomly pick the answer from the library
+    library = ["cat", "dog", "mouse", "letter", "elephant", "bottomless", "house", "polecat", "coffee", "apple"]
+    @answer = library.sample
+
+    return @answer
```

Comment:
Since `@answer` is being returned here and set = to `@answer`, you should use a local variable here rather than an instance variable. You only need the instance variable on line 4 where the result of this method is set.